lms/static/js/customwmd.js

```diff
@@ -0,0 +1,247 @@
+// Once generated by CoffeeScript 1.9.3, but now lives as pure JS
+/* eslint-disable */
+// TODO: Examine all of the xss-lint exceptions
```

Comment:
is this TODO still relevant? If so, I'd advocate for filing a ticket, somewhere, and mentioning it here

Reply:
Yeah I added the TODO manually here. Previous code had a blanket exception for static/coffee in the xss-linter. So I made this comment because I wanted to just keep ignoring the issue for this PR, but note it for future generations.

Yeah I can make a ticket. But we have a lot of extent xss issues already (that aren't even covered by lint comments). Not sure that'll really help it get looked at. But it's a start I guess.

Reply:
https://openedx.atlassian.net/browse/PLAT-2084 -- will add comments to these files.